README.md

```diff
@@ -16,6 +16,13 @@ Second, read the [Troubleshooting Checklist](https://github.com/Homebrew/brew/bl
 
 **If you don't read these it will take us far longer to help you with your problem.**
 
+## Contributing
+We'd love you to contribute to Homebrew. First, Please read our [Contribution guide](https://github.com/Homebrew/brew/blob/master/.github/CONTRIBUTING.md) and [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct).
+
+We explicitly welcome contributions from people who have never contributed to open-source before: we were all beginners once! We can help take a partially working pull request and help you get it merged. We are also actively seeking to diversify our contributors and especially welcome contributions from women from all backgrounds and people of colour.
```

Comment:
Also feels awkward to me, given this in some way limits diversity to gender and color, despite there being a lot more axes in the diversity spectrum. Maybe something like “[…] and especially welcome contributions from people underrepresented in tech/IT”? Sadly, I don't have a better idea on how to improve the wording.